mp/src/game/client/momentum/ui/SettingsPanel/OnlineSettingsPage.cpp

```diff
@@ -21,62 +21,30 @@ OnlineSettingsPage::OnlineSettingsPage(Panel* pParent) : BaseClass(pParent, "Onl
     m_pEnableGhostTrails->AddActionSignalTarget(this);
     m_pEnableColorAlphaOverride = new CvarToggleCheckButton(this, "EnableAlphaOverride", "#MOM_Settings_Override_Alpha_Enable", "mom_ghost_online_alpha_override_enable");
     m_pEnableColorAlphaOverride->AddActionSignalTarget(this);
-    m_pAlphaOverrideSlider = new CvarSlider(this, "AlphaOverrideSlider", nullptr, 0.0f, 255.0f, "mom_ghost_online_alpha_override");
+    m_pAlphaOverrideSlider = new CvarSlider(this, "AlphaOverrideSlider", nullptr, 0.0f, 255.0f, "mom_ghost_online_alpha_override", false, true);
     m_pAlphaOverrideSlider->AddActionSignalTarget(this);
-    m_pAlphaOverrideInput = new TextEntry(this, "AlphaOverrideEntry");
+    m_pAlphaOverrideInput = new CvarTextEntry(this, "AlphaOverrideEntry", "mom_ghost_online_alpha_override");
     m_pAlphaOverrideInput->SetAllowNumericInputOnly(true);
     m_pAlphaOverrideInput->AddActionSignalTarget(this);
 
     LoadControlSettings("resource/ui/SettingsPanel_OnlineSettings.res");
 }
 
-void OnlineSettingsPage::OnApplyChanges()
-{
-    BaseClass::OnApplyChanges();
-
-    
-}
-
 void OnlineSettingsPage::LoadSettings()
 {
-
-    UpdateSliderSettings();
-}
-
-void OnlineSettingsPage::OnCheckboxChecked(Panel* p)
-{
-}
-
-void OnlineSettingsPage::OnTextChanged(vgui::Panel* panel)
-{
-    BaseClass::OnTextChanged(panel);
-
-    if (panel == m_pAlphaOverrideInput)
+    if (m_pEnableColorAlphaOverride && m_pAlphaOverrideSlider && m_pAlphaOverrideInput)
```

Comment:
Same here, these pointers shouldn't be null, they're made on construction of the panel